.env

```diff
@@ -1,3 +1,3 @@
-KNUU_TIMEOUT=60m # The timeout for the tests
+KNUU_TIMEOUT=120m   # The timeout for the tests
 KNUU_BUILDER=docker # The builder to use for building images. docker or kubernetes
-LOG_LEVEL=info # The debug level. debug, info, warn, error
+LOG_LEVEL=info      # The debug level. debug, info, warn, error
```

Comment:
this one also is a parameter in knuu object and user can set it however they want